docs/rst/manual/_ext/ert_jobs.py

```diff
@@ -0,0 +1,198 @@
+        config_section_node.append(job_config_title_node)
+        config_section_node.append(job_config_text_node)
+        node.append(config_section_node)
+
+    def _add_job_details(self, node):
+        job_namespace_node = nodes.definition_list_item(
+            "",
+            nodes.term("", "", nodes.strong(text="Namespace")),
+            nodes.definition("", nodes.strong(text=self.namespace)),
+        )
+        job_package_node = nodes.definition_list_item(
+            "",
+            nodes.term("", "", nodes.strong(text="Source package")),
+            nodes.definition("", nodes.strong(text=self.job_source)),
+        )
+        job_details_node = nodes.definition_list(
+            "", job_namespace_node, job_package_node
+        )
+        node.append(job_details_node)
+
+    def _add_job_description_section(self, node, state):
+        _parse_raw_rst(self.description, node, state)
+        self._add_job_details(node)
+
+    def _add_example_section(self, node, state):
+        example_title_node = nodes.title(text="Examples")
+
+        example_section_node = nodes.section(ids=[self.name + "-examples"])
+        example_section_node.append(example_title_node)
+
+        _parse_raw_rst(self.examples, example_section_node, state)
+
+        node.append(example_section_node)
+
+    def _add_argparse_section(self, node, state):
+        argparse_title_node = nodes.title(text="Script arguments")
+
+        argparse_section_node = nodes.section(ids=[self.name + "-argparse"])
+        argparse_section_node.append(argparse_title_node)
+
+        _parse_raw_rst(argparse_template_text, argparse_section_node, state)
+
+        node.append(argparse_section_node)
+
+    def add_to_node(self, root_node, state):
+        node = self._create_job_node()
+        self._add_job_description_section(node, state)
+        self._add_job_config_section(node)
+        if self.argparse is not None:
+            self._add_argparse_section(node, state)
+        if self.examples:
+            self._add_example_section(node, state)
+        root_node.append(node)
+
+
+class ErtJobs(SphinxDirective):
+
+    has_content = True
+
+    def _generate_namespace_sections(self, jobs):
+
+        sections = dict()
+        for job_name, docs in jobs.items():
+            if job_name.islower():
+                continue
+
+            namespace = docs.get("namespace", "other")
+
+            section = namespace.split(".")[0]
+            if section not in sections:
+                sections[section] = list()
+
+            sections[section].append(
+                JobDocumentation(
+                    name=job_name,
+                    namespace=namespace,
+                    job_source=docs.get("source_package", "PACKAGE NOT PROVIDED"),
+                    description=docs.get("description", ""),
+                    job_config_file=docs.get("config_file", "No config file provided"),
+                    argparse=docs.get("arg_parse", None),
```

Comment:
Why is `argparse` set to `None` and `description` to `""` by default? Are they not both mean to be `rst`?

Reply:
The intention was to allow the jobs to be documented automatically by providing their argparser, however, after some discussion we decided not to go down that route, so should be removed!